user_tools/src/spark_rapids_pytools/resources/onprem-configs.json

```diff
@@ -1,26 +1,30 @@
 {
   "dependencies": {
     "deployMode": {
-      "LOCAL": [
-        {
-          "name": "Apache Spark",
-          "uri": "https://archive.apache.org/dist/spark/spark-3.5.0/spark-3.5.0-bin-hadoop3.tgz",
-          "type": "archive",
-          "relativePath": "jars/*",
-          "sha512": "8883c67e0a138069e597f3e7d4edbbd5c3a565d50b28644aad02856a1ec1da7cb92b8f80454ca427118f69459ea326eaa073cf7b1a860c3b796f4b07c2101319",
-          "size": 400395283
-        }
-      ],
-      "SPARK333-LOCAL": [
-        {
-          "name": "Apache Spark",
-          "uri": "https://archive.apache.org/dist/spark/spark-3.3.3/spark-3.3.3-bin-hadoop3.tgz",
-          "type": "archive",
-          "relativePath": "jars/*",
-          "sha512": "ebf79c7861f3120d5ed9465fdd8d5302a734ff30713a0454b714bbded7ab9f218b3108dc46a5de4cc2102c86e7be53908f84d2c7a19e59bc75880766eeefeef9",
-          "size": 299426263
-        }
-      ]
+      "LOCAL": {
+        "//activeBuildVer": "Define this key in order to set the default buildVer for that platform",
+        "activeBuildVer": "331",
```

Comment:
nit: Will this cause failure? Since we do not have entry for `331`?

Reply:
My bad. I used that to test and I did not notice that I did `git add --all` 
Removed.